pkg/security/podsecuritypolicy/util/util.go

```diff
@@ -182,8 +183,9 @@ func PSPAllowsHostVolumePath(psp *extensions.PodSecurityPolicy, hostPath string)
 		return true
 	}
 
+	path := filepath.Clean(hostPath)
```

Comment:
I would expect this to be a bug on storage allowing `..` vs something wrong with PSP